python/pyspark/pandas/groupby.py

```diff
@@ -805,7 +874,7 @@ def all(self, skipna: bool = True) -> FrameLike:
         5  False
         """
         groupkey_names = [SPARK_INDEX_NAME_FORMAT(i) for i in range(len(self._groupkeys))]
-        internal, sdf = self._prepare_reduce(groupkey_names)
+        internal, _, sdf = self._prepare_reduce(groupkey_names)
```

Comment:
Hm, I can't follow this change. `_prepare_reduce` seems returning internal frame and dataframe. what's the one in the middle?